main/lsp/wrapper.h

```diff
@@ -6,14 +6,23 @@
 #include "lsp.h"
 #include "main/lsp/LSPMessage.h"
 #include "spdlog/sinks/stdout_color_sinks.h"
-#include <sstream>
 #include <string_view>
 namespace sorbet::realmain::lsp {
 
 class LSPWrapper {
 private:
     static const std::string EMPTY_STRING;
 
+    class LSPOutputToVector : public LSPOutput {
```

Comment:
can this class be defined in `cc` file? is it ever used by other bazel modules?

Reply:
It's stored in `LSPWrapper`, so I think I need to have it defined here so the compiler can calculate the size of `LSPWrapper`.

Reply:
You are right that it's not used by any other bazel modules.

Reply:
> It's stored in LSPWrapper,

can we store a `unique_ptr< LSPOutputToVector>` to sidestep this?

Reply:
Sure; I'll forward decl it